app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java

```diff
@@ -60,6 +60,9 @@
     private BitmapDescriptor currentPOIIcon;
     private Location currentLocation;
 
+    //EV centerCoordinates
+    private LatLng initialZoomLocation = new LatLng(45.495638 ,-73.578258);
```

Comment:
Why is this the default? Isn't it supposed to be Hall?

Reply:
default was always EV

Reply:
mmm Do you think that we should put EV as the "default" currentLocation when we calculate distances b/w POIs too?

Reply:
could we add those to class constant just so it is clear which coordinate we are using?

Reply:
done tarek

Reply:
and Viveka, no, I don't think so